modules/exploits/linux/http/hadoop_unauth_exec.rb

```diff
@@ -26,8 +26,9 @@ def initialize(info = {})
       'References'     =>
         [
           ['URL', 'http://archive.hack.lu/2016/Wavestone%20-%20Hack.lu%202016%20-%20Hadoop%20safari%20-%20Hunting%20for%20vulnerabilities%20-%20v1.0.pdf'],
-          ['URL', 'https://github.com/vulhub/vulhub/tree/master/hadoop/unauthorized-yarn']
-          # Note, there will never be a CVE for this issue, unless something radical changes in the CVE inclusion rules.
+          ['URL', 'https://github.com/vulhub/vulhub/tree/master/hadoop/unauthorized-yarn'],
+          # Note, there will never be a CVE for this issue, since this is a misconfiguration by the administrator rather than a vulnerability in the software:
+          ['URL', 'https://hadoop.apache.org/docs/current/hadoop-project-dist/hadoop-common/SecureMode.html']
```

Comment:
This is a nice addition but it fails to explain how this ties into the rest of this exploit. All it tells a user is "use Kerberos" without explaining how the default settings are bad. It also doesn't tell me how switching to Kerberos helps things. I'd update the comment here to explain this a bit more.